dags/glam.py

```diff
@@ -66,7 +66,7 @@
     task_id="wait_for_copy_deduplicate_main_ping",
     external_dag_id="copy_deduplicate",
     external_task_id="copy_deduplicate_main_ping",
-    execution_delta=timedelta(hours=1),
+    execution_delta=timedelta(hours=16),
```

Comment:
```suggestion
    execution_delta=timedelta(hours=12),
```
I think the delta is 12h here? `copy_deduplicate` runs at 1am

Reply:
:facepalm: I got the timezones confused.